todo_app/data/trello_items.py

```diff
@@ -8,6 +8,20 @@ def __init__(self, id, status, title ):
         self.name = title
         self.status = status
 
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+    @property
+    def items(self):
+        return self._items
+    @property
+    def todoitems(self):
+        cards = []
+        for todoitem in self._items:
+            if todoitem.status == '60c73323b5d71d7a754a20fc':
```

Comment:
Having this id hardcoded makes the app less flexible - even though it's not a secret it's still well worth extracting this out into a variable;
```suggestion
            if todoitem.status == os.getenv('TODO_LIST'):
```
That way we can deploy this application to multiple environments talking to different Trello boards (and also makes our tests safer; there's reduced risk they accidentally talk to our production environment!)